src/app/shell/personal-cabinet/provider/create-provider-admin/create-provider-admin.component.html

```diff
@@ -0,0 +1,115 @@
+        [invalid]="ProviderAdminFormGroup.get('middleName').invalid && ProviderAdminFormGroup.get('middleName').touched"
+        [forbiddenCharacter]="ProviderAdminFormGroup.get('middleName').errors?.pattern?.requiredPattern">
+      </app-validation-hint-for-input>
+      <app-validation-hint-for-input type="validLength"
+        [minLength]="ProviderAdminFormGroup.get('middleName').errors?.minlength"
+        [maxLength]="ProviderAdminFormGroup.get('middleName').errors?.maxlength"
+        [invalid]="ProviderAdminFormGroup.get('middleName').invalid && ProviderAdminFormGroup.get('middleName').touched"
+        [minCharachters]="1" [maxCharachters]="30">
+      </app-validation-hint-for-input>
+
+      <label class="step-label"> Телефон <span class="step-required">*</span></label>
+      <mat-form-field>
+        <div fxLayout='row' fxLayoutAlign='start'>
+          <span class="step-text">{{constants.PHONE_PREFIX}}</span>
+          <input matInput class="step-input" type="tel" formControlName="phoneNumber" appDigitOnly ui-number-mask
+            ui-hide-group-sep maxlength="{{constants.PHONE_LENGTH}}">
+        </div>
+      </mat-form-field>
+      <app-validation-hint-for-input type="requiredEmptyField"
+        [isEmptyCheck]="ProviderAdminFormGroup.get('phoneNumber').errors?.required"
+        [invalid]="ProviderAdminFormGroup.get('phoneNumber').invalid && ProviderAdminFormGroup.get('phoneNumber').touched">
+      </app-validation-hint-for-input>
+      <app-validation-hint-for-input type="minDigitsLength"
+        [minLength]="ProviderAdminFormGroup.get('phoneNumber').errors?.minlength"
+        [invalid]="ProviderAdminFormGroup.get('phoneNumber').invalid && ProviderAdminFormGroup.get('phoneNumber').touched"
+        [minCharachters]="ProviderAdminFormGroup.get('phoneNumber').errors?.minlength?.requiredLength">
+      </app-validation-hint-for-input>
+
+      <label class="step-label"> Емейл <span class="step-required">*</span></label>
+      <mat-form-field>
+        <input matInput class="step-input" type="email" formControlName="email">
+      </mat-form-field>
+      <app-validation-hint-for-input type="requiredEmptyField"
+        [isEmptyCheck]="ProviderAdminFormGroup.get('email').errors?.required"
+        [invalid]="ProviderAdminFormGroup.get('email').invalid && ProviderAdminFormGroup.get('email').touched">
+      </app-validation-hint-for-input>
+      <app-validation-hint-for-input type="validEmail"
+        [isEmailCheck]="ProviderAdminFormGroup.get('email').errors?.email"
+        [invalid]="ProviderAdminFormGroup.get('email').invalid && ProviderAdminFormGroup.get('email').touched">
+      </app-validation-hint-for-input>
+
+      <ng-container *ngIf="(!isDeputy)">
+        <label class="step-label"> Гуртки <span class="step-required">*</span></label>
+        <app-workshop-checkbox-dropdown [workshops]="workshops$ | async" (workshopCheck)="onWorkshopsSelect($event)">
+        </app-workshop-checkbox-dropdown>
```

Comment:
Here we should restyle it a bit because the whole dropdown cannot be viewed on the screen nevertheless screen size
![image](https://user-images.githubusercontent.com/27493959/158139498-376f5e3a-54af-4d77-8619-84f0e6944f99.png)
![image](https://user-images.githubusercontent.com/27493959/158139532-85afa11c-03a3-4590-95d3-016143e6b19a.png)

Reply:
has updated